docs/manage/remotes.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Parts, Sub-Parts and Remotes"
+linkTitle: "Remotes and Sub-Parts"
+weight: 40
+type: "docs"
+description: "Connect robots to each other."
+tags: ["server", "components", "services"]
+---
+
+When robots can communicate with each other, they can share resources and operate collaboratively.
+This document explains how to establish secure connections between robots.
+
+### Robot parts
+
+Robots are organized into *parts*, where each part represents a computer (a single-board computer like a Raspberry Pi or a desktop, laptop, or other computer) running `viam-server`, the hardware [components](/components/) attached to it, and any [services](/services/) or other resources running on it.
+Every robot has a main part.
+Multi-part robots also have one or more *sub-parts* representing additional computers running `viam-server`.
+
+There are two ways to link robot parts:
+
+- **Sub-part**: If you have two computers within the *same robot*, use one as the main part and [connect the other to it as a sub-part](#configure-a-sub-part).
+
+- **Remote**: To connect two computers that are parts of *different robots*, [add one robot part as a remote part of the other robot](#configure-a-remote).

```

Comment:
Within robot parts, this makes sense calling it remotes. We are still missing one slight thing which is to just talk about a remote that is either running on some non viam-server SDK or a viam-server running outside of App. We don't have to make large mention of them, but it is important to spell it out in one of our `###`s.

Reply:
I've gotten mixed messages about whether we are supposed to talk about running viam-server without connecting to the app.

Sorry what does ### mean? Does it mean in some doc somewhere or in a section here?

I'm not super confident in my understanding of how else you would use it or how much of an edge case that is. Should I include a note box at the bottom of "Configure a Remote" that says
"You can also configure a remote to:
 - An instance of `viam-server` that is not connected to the Viam app
 - A server other than `viam-server`"
 but then would we need to include instructions for how? How do they find the address and secret?

Reply:
hmm. maybe let's just keep it as is for now and get feedback instead. I don't have any great ideas for how to section it up better right now